scripts/diagnose.sh

```diff
@@ -147,16 +147,17 @@ commands=(
 	'sysctl -a'
 	'systemctl list-units --failed --no-pager'
 	'top -b -n 1'
-	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
+	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/balena-supervisor/supervisor.conf|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
 
 	# SUPERVISOR specific commands
 	'echo === SUPERVISOR ==='
-	'$ENG exec resin_supervisor cat /etc/resolv.conf'
+	'$ENG exec $($ENG ps | awk "/supervisor/{print \$1}") cat /etc/resolv.conf'
+	'$ENG logs balena_supervisor'
```

Comment:
```suggestion
 	'$ENG logs $($ENG ps | awk "/supervisor/{print \$1}")'
```

sorry :)

Reply:
I got this one before you posted it, but I still don't think I'm making PAR for this course